ReactiveCocoa/Swift/UnidirectionalBinding.swift

```diff
@@ -0,0 +1,315 @@
+	@discardableResult
+	public static func <~ <Source: SignalProtocol where Source.Value == ValueType.Wrapped, Source.Error == NoError>(consumer: Self, signal: Source) -> Disposable {
+		return consumer <~ signal.map(ValueType.init(reconstructing:))
+	}
+
+	/// Creates a signal from the given producer, which will be immediately bound to
+	/// the given property, updating the property's value to the latest value sent
+	/// by the signal.
+	///
+	/// ````
+	/// let property = MutableProperty(0)
+	/// let producer = SignalProducer<Int, NoError>(value: 1)
+	/// property <~ producer
+	/// print(property.value) // prints `1`
+	/// ````
+	///
+	/// ````
+	/// let property = MutableProperty(0)
+	/// let producer = SignalProducer({ /* do some work after some time */ })
+	/// let disposable = (property <~ producer)
+	/// ...
+	/// // Terminates binding before property dealloc or
+	/// // signal's `completed` event.
+	/// disposable.dispose()
+	/// ````
+	///
+	/// - note: The binding will automatically terminate when the property is
+	///         deinitialized, or when the created producer sends a `completed`
+	///         event.
+	///
+	/// - parameters:
+	///   - consumer: A property to bind to.
+	///   - producer: A producer to bind.
+	///
+	/// - returns: A disposable that can be used to terminate binding before the
+	///            deinitialization of property or producer's `completed` event.
+	@discardableResult
+	public static func <~ <Source: SignalProducerProtocol where Source.Value == ValueType.Wrapped, Source.Error == NoError>(consumer: Self, producer: Source) -> Disposable {
+		return consumer <~ producer.map(ValueType.init(reconstructing:))
+	}
+
+	/// Binds `destinationProperty` to the latest values of `sourceProperty`.
+	///
+	/// ````
+	/// let dstProperty = MutableProperty(0)
+	/// let srcProperty = ConstantProperty(10)
+	/// dstProperty <~ srcProperty
+	/// print(dstProperty.value) // prints 10
+	/// ````
+	///
+	/// ````
+	/// let dstProperty = MutableProperty(0)
+	/// let srcProperty = ConstantProperty(10)
+	/// let disposable = (dstProperty <~ srcProperty)
+	/// ...
+	/// disposable.dispose() // terminate the binding earlier if
+	///                      // needed
+	/// ````
+	///
+	/// - note: The binding will automatically terminate when either property is
+	///         deinitialized.
+	///
+	/// - parameters:
+	///   - consumer: A property to bind to.
+	///   - property: A property to bind.
+	///
+	/// - returns: A disposable that can be used to terminate binding before the
+	///            deinitialization of destination property or source property
+	///            producer's `completed` event.
+	@discardableResult
+	public static func <~ <Source: PropertyProtocol where Source.Value == ValueType.Wrapped>(consumer: Self, property: Source) -> Disposable {
+		return consumer <~ property.producer
+	}
+}
+
+/// A type-erased view of a binding consumer.
+public class AnyBindingConsumer<Value>: BindingConsumer {
```

Comment:
It is made for all non-KVO compliant properties/keypaths, which are all currently in Rex.